app/assets/v2/js/metamask-approval.js

```diff
@@ -23,7 +40,20 @@ async function metamaskApproval() {
     } catch (error) {
       _alert('Permission to connect to metamask rejected. Allow gitcoin to connect to metamask.', 'warning');
     }
+  } else if (window.web3) {
+    injectScript('https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.0.0-beta.34/dist/web3.min.js')
```

Comment:
i think well want to inject 0.20.3 not 1.0.0 right since the site uses 0.20.3

Reply:
Good question, I don't know what are all the diff but if we can jump to the last version I believe we be covering more actual and future browsers.